docs/core/diagnostics/observability-with-otel.md

```diff
@@ -0,0 +1,429 @@
+```
+
+This output tells you it's listening on `0.0.0.0:4317`, so you can configure that port as the destination for your OTLP exporter.
+
+Open the `AppSettings.json` file for our project, and add the following line, changing the port if applicable.
+
+``` json
+"OTLP_ENDPOINT_URL" :  "http://localhost:4317/"
+```
+
+Restart the greeter process so that it can pick up the property change and start directing tracing information to Jaeger.
+
+Now, you should be able to see the Jaeger UI at [`http://localhost:16686/`](http://localhost:16686/) from a web browser.
+
+[![Jaeger query for traces](./media/jaeger-search-results.thumb.png)](./media/jaeger-search-results.png#lightbox)
+
+To see a list of traces, select `OTel-Prometheus-grafana-Yaeger` from the **Service** dropdown. Selecting a trace should show a gant chart of the activities as part of that trace. Clicking on each of the operations shows more details about the activity.
+
+[![Jaeger Operation Details](./media/jaeger-activity-details.thumb.png)](./media/jaeger-activity-details.png#lightbox)
+
+In a distributed system, you want to send traces from all processes to the same Jaeger installation so that it can correlate the transactions across the system.
+
+You can make your app a little more interesting by having it make HTTP calls to itself.
+
+- Add an `HttpClient` factory to the application
+
+   :::code language="csharp" source="snippets/OTel-Prometheus-Grafana-Yaeger/csharp/Program.cs" id="Snippet_HttpClientFactory":::
+
+- Add a new endpoint for making nested greeting calls
+
+   :::code language="csharp" source="snippets/OTel-Prometheus-Grafana-Yaeger/csharp/Program.cs" id="Snippet_MapNested":::
+
+- Implement the endpoint so that it makes HTTP calls that can also be traced. In this case, it calls back to itself in an artificial loop (really only applicable to demo scenarios).
+
+   :::code language="csharp" source="snippets/OTel-Prometheus-Grafana-Yaeger/csharp/Program.cs" id="Snippet_SendNestedGreeting":::
+
+This results in a more interesting graph with a pyramid shape for the requests, as each level waits for the response from the previous call.
+
+[![Jaeger nested dependency results](./media/jaeger-nested-activity-details.thumb.png)](./media/jaeger-nested-activity-details.png#lightbox)
+
+## Example: Use Azure Monitor and Application Insights
+
+In the previous example, you used separate open-source applications for metrics and tracing. There are many commercial APM systems available to choose from. In Azure, the primary application-monitoring product in is Application Insights, which is part of Azure Monitor.
+
+One of the advantages of an integrated APM product is that it can correlate the different observability data sources. To make the ASP.NET experience with Azure Monitor easier, they've created a wrapper package that does most of the heavy lifting of configuring OpenTelemetry.
```

Comment:
```suggestion
One of the advantages of an integrated APM product is that it can correlate the different observability data sources. To make the ASP.NET experience with Azure Monitor easier, a wrapper package is provided which does most of the heavy lifting of configuring OpenTelemetry.
```

Reply:
@rajkumar-rangaraj please help the wording here.